app/screens/ExploreScreen/ExploreScreen.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { FC } from "react"
+import {
+  ActivityIndicator,
+  Image,
+  ImageSourcePropType,
+  ImageStyle,
+  SectionList,
+  SectionListData,
+  TextStyle,
+  View,
+  ViewStyle,
+} from "react-native"
+import { Button, Carousel, Screen, Text } from "../../components"
+import { TabScreenProps } from "../../navigators/TabNavigator"
+import { colors, spacing } from "../../theme"
+import { useHeader } from "../../hooks/useHeader"
+import { translate } from "../../i18n"
+import { useRecommendations } from "../../services/api"
+import { WEBFLOW_MAP } from "../../services/api/webflow-consts"
+import { groupBy } from "../../services/api/webflow-helpers"
+import { RawRecommendations } from "../../services/api/webflow-api.types"
+import { openLinkInBrowser } from "../../utils/openLinkInBrowser"
+
+const exploreMap = require("../../../assets/images/exploreMap.png")
+
+interface CreditData {
+  author: string
+  text: string
+  link: string
+}
+interface ExploreMapProps {
+  image: ImageSourcePropType
+  description: string
+  credit: CreditData
+}
+
+const recommendationTypes = Object.values(WEBFLOW_MAP.recommendationType)
+type RecommendationType = typeof recommendationTypes[number]
+type GroupedRecommendations = Record<RecommendationType, RawRecommendations[]>
+
+const initialRecs = recommendationTypes.reduce<GroupedRecommendations>(
+  (acc, recommendationType) => ({ ...acc, [recommendationType]: [] }),
+  {} as GroupedRecommendations,
+)
+
+const FoodAndDrink = ({ item }) => {
+  return <Carousel preset="dynamic" data={item} />
+}
+
+const ArtMurals = ({ item }) => {
+  return (
+    <View style={$carousel}>
+      <Carousel preset="dynamic" data={item} />
+      <View style={$artButtonContainer}>
+        <Button
+          text="Open Google Maps list"
+          onPress={() =>
+            openLinkInBrowser(
```

Comment:
Are all the art murals in the same location, or is this something that will be dynamic per item later?

Reply:
So this is going to be a list of spots that Jenna will add to a google map list so it should be just open that link :)